_posts/2018-07-25-the_incredibles_3.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "The Incredibles 3"
+date: "2018-07-25 21:12:49 -0500"
+---
+
+I've been thinking a lot about _The Incredibles 2_ since I saw it. Now that we
+have two entries in the series, we can make some statements about what subjects
+the franchise as a whole addresses. At least if we're comfortable extrapolating
+a line from two points. I'm comfortable with doing that and so I'd like to talk
+about some of those topics and how I'd like to see them developed in a
+theoretical third movie: _The Incredibles 3_.
+
+
+## Content Warning
+
+Expect spoilers for _The Incredibles_ and _The Incredibles 2_ below. Not a ton,
+but still.
+
+Also, I'm doing a somewhat feminist reading of the franchise and below I'm going
+to lightly touch on racism and situations where men emotionally manipulate
+women. I won't go into detail, but if you feel like you don't want to deal with
+either of those right now, this is your chance to bail on reading this post. I
+mean—you can actually bail any time. I can't stop you from closing the browser
+window whenever you want. I'm not your real dad. And he can't stop you, either,
+probably.
+
+
+## Two Brief Tangents
+
+Before we get into my thoughts, I wanted to mention some things about _The
+Incredibles 2_ that are sort of indirectly related to my reading. Other people
```

Comment:
"that relate indirectly" - it still has an adverb but uses active voice.

Reply:
Oh man. This was super confusing to me at first. You're just giving the common writing advice to avoid the passive voice?